MD/md-20/README.md

```diff
@@ -0,0 +1,93 @@
+# MD-20: Bridge Swap Mechanism
+
+**Description**: This MD outlines and proposes the mechanics for the swap to take place, as part of the lock-mint-swap Bridge Transfer.
+
+## Overview
+
+The Bridge uses a lock-mint-swap process to provide our users with native gas-paying L2 $MOVE for their locked L1 ERC20 $MOVE. This transfer occurs in one sequence for the user, within the timelock period. The Lock and Mint phases of the sequence have already been implemented and were proposed in [RFC40](https://github.com/movementlabsxyz/rfcs/tree/main/0040-atomic-bridge). Therefore, these parts of the sequence will not be discussed here, only referenced.
+
+The total gas required on the initiator side and the computation of this amount will also not be discussed here, as it has already been explored in [MD-17](https://github.com/movementlabsxyz/MIP/blob/6722c67a8434de07c6612e46b5a023b63ad8dcbd/MD/md-17/README.md). This MD focuses solely on the swap step of L2 $MOVE. When a user transfers from L2 back to L1, the [AtomicBridgeCounterpartyMOVE.sol](https://github.com/movementlabsxyz/movement/blob/andygolay/atomic-bridge-initiator-move/protocol-units/bridge/contracts/src/AtomicBridgeCounterpartyMOVE.sol) simply withdraws the ERC20 $MOVE from the pool and issues it back to the user. Since ERC20 is not a native gas-paying asset like ETH, no swap is needed on L1 in this direction. However, a swap will occur on L2.
+
+To summarize:
+- **L1 -> L2**: Lock->Mint->Swap
+- **L2 -> L1**: Swap->Lock->Mint
```

Comment:
if this term is inaccurate and the only reason is that accross the organization it used despite being wrong, we should not proceed to use it but fix the issue.